web-services/accumulo/pom.xml

```diff
@@ -69,17 +69,42 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>io.astefanutti.metrics.cdi</groupId>
-            <artifactId>metrics-cdi</artifactId>
```

Comment:
I haven't verified yet, but I'm pretty sure this change is the reason that the web service deployment is failing in the test runner. That lib is needed for CDI injection of the dropwizard metrics-annotated fields.

From server.log...
```
org.jboss.weld.exceptions.DeploymentException: WELD-001408: Unsatisfied dependencies for type Counter with qualifiers @Default
  at injection point [BackedAnnotatedField] @Inject @Metric private datawave.webservice.operations.remote.RemoteAccumuloService.failureCounter
  at datawave.webservice.operations.remote.RemoteAccumuloService.failureCounter(RemoteAccumuloService.java:0)
```
Sadly, Wildfly doesn't give anything useful like a ClassNotFoundException in this case  :-(

Reply:
Thanks! Yeah I had this figured out around the time you posted this. Needed a few other things as well, but it's back to building proper now.